lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -614,9 +614,11 @@ def create_test_target_copy_resources_script(test_spec)
                 pod_target.resource_paths.values_at(*spec_paths_to_include).flatten.compact
               end
             end
-            generator = Generator::CopyResourcesScript.new(resource_paths_by_config, target.platform)
-            update_changed_file(generator, path)
-            add_file_to_support_group(path)
+            unless resource_paths_by_config.values.all?(&:empty?)
```

Comment:
nit: `.each_value.all?(&:empty?)` to avoid the intermediary array

Reply:
nice will update.